### PR TITLE
[Update Plugin] Update ReadAllButton

### PR DESCRIPTION
**Pull request by KrstlSkll69**
This should update the "read all button", so its not so ugly and bland. Should be able to work with themes (especially the client theme plugin)

### CODE REVIEW

**Comment by KrstlSkll69** [comment]
updated "Constant.ts" twice to fix typo i made

File: src/plugins/readAllNotificationsButton/style.css [hunk 1]
```diff
@@ -0,0 +1,9 @@
+
+/* recolour and reshape Read All Button*/
+[class^=scroller__] > button:first-of-type {
```
Comment (Vendicated) [inline src/plugins/readAllNotificationsButton/style.css:3]: instead of using this insane selector, give the button a class and select that instead :3

**Comment by KrstlSkll69** [comment]
@Vendicated ok changed (I think, idk ive never used github before so im trying)

File: src/utils/constants.ts [hunk 2]
```diff
@@ -425,7 +425,11 @@ export const Devs = /* #__PURE__*/ Object.freeze({
     newwares: {
         name: "newwares",
         id: 421405303951851520n
-    }
+    },
+    KrystalSkull: {
+        name: "krystalskulloffical"
+        id: 929208515883569182
+
```
Comment (MrDiamondDog) [inline src/utils/constants.ts:432]: me when syntax error

File: src/plugins/readAllNotificationsButton/style.css [hunk 3]
```diff
@@ -0,0 +1,14 @@
+/* recolour and reshape Read All Button*/
+.vc-readAllNotis-button {
+    color: var(--interactive-normal) !important; /* text color */
+    background-color: var(--background-tertiary) !important; /* button color */
+    border-radius: 5px !important; /* increase radius for "more roundness" */
+    margin: 2px 0 8px 9px /* original styles */
+}
+
+.vc-readAllNotis-button:hover {
+  background-color: var(--brand-experiment) !important;
```
Comment (MrDiamondDog) [inline src/plugins/readAllNotificationsButton/style.css:10]: make sure styling is consistent (tab width is different in this block vs. the one above)

**Review by MrDiamondDog — CHANGES_REQUESTED** [review]

File: src/utils/constants.ts (hunk 2 above)
Comment (KrstlSkll69) [reply src/utils/constants.ts:432]: OMFG ITS THAT GODDAMN TYPO I THOUGHT I FIXXED THAT THING BEFORE

File: src/utils/constants.ts [hunk 4]
```diff
@@ -425,7 +425,11 @@ export const Devs = /* #__PURE__*/ Object.freeze({
     newwares: {
         name: "newwares",
         id: 421405303951851520n
-    }
+    },
+    KrystalSkull: {
+        name: "krystalskulloffical",
+        id: 929208515883569182n
+
```
Comment (MrDiamondDog) [inline src/utils/constants.ts:432]:
```suggestion
    }
```

**Review by MrDiamondDog — CHANGES_REQUESTED** [review]

File: src/utils/constants.ts (hunk 4 above)
Comment (KrstlSkll69) [reply src/utils/constants.ts:432]: fuck, this thing keeps making typos just to hurt me on purpose

**Comment by KrstlSkll69** [comment]
misclicked didn't mean to close

**Comment by KrstlSkll69** [comment]
@Vendicated bump

**Comment by KrstlSkll69** [comment]
@Vendicated bump

**Comment by Vendicated** [comment]
cat-patience-cat

**Comment by Sqaaakoi** [comment]
i was half way through writing a comment, but yes anyway

please get some patience. vee has a life outside of vencord

**Comment by Vendicated** [comment]
@CodiumAI-Agent /review

**Comment by KrstlSkll69** [comment]
reopening this because i only opened this as it was what Vee told me to do, but i dont know what to do now nor do i care, i did how i knew how to do it, might try to however they want ig we'll see

**Comment by Vendicated** [comment]
superseded by https://github.com/Vendicated/Vencord/commit/8ab56f5bcfe720b8e6d03f85b5f4ada1a18bfc37

**Pull request closed without merging**